mbtiles/src/metadata.rs

```diff
@@ -136,6 +136,11 @@ impl Mbtiles {
                         tj.other.insert(name, Value::String(value));
                     }
                     "agg_tiles_hash" => agg_tiles_hash = Some(value),
+                    "scheme" => {
+                        if value != "tms" {
+                            warn!("File {file} has an unexpected metadata value schema='{value}'. Only 'tms' is supported. Ignoring.")
```

Comment:
Typo, lets use a var instead

```suggestion
                            warn!("File {file} has an unexpected metadata value {name}='{value}'. Only 'tms' is supported. Ignoring.")
```